src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2404,7 +2416,7 @@ void processContextElement
     /* All the attributes existing in the request are added to the response with 'found' set to false
      * in the of UPDATE/DELETE and true in the case of APPEND
      */
-    bool foundValue = (strcasecmp(action.c_str(), "append") == 0);
+    bool foundValue = (strcasecmp(action.c_str(), "append") == 0) || (strcasecmp(action.c_str(), "appendonly") == 0);
```

Comment:
What if we do this (expand to v1) in a separate PR?
That and more functests ...